core/invoker/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -717,13 +731,15 @@ class ContainerPool(
      * Actually deletes the containers.
      */
     private def teardownContainer(container: Container)(implicit transid: TransactionId) = try {
-        val size = this.getLogSize(container, !standalone)
-        val rawLogBytes = container.synchronized {
-            this.getDockerLogContent(container.containerId, 0, size, !standalone)
+        if (saveContainerLog) {
```

Comment:
I assume we only record activation logs to db and never to disk in production? So will only be true during tests? This should increase performance quite a bit.